<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<title>Propra G4</title>
<style type="text/css" media="screen">
body { background: #e7e7e7; font-family: Verdana, sans-serif; font-size: 11pt; }
#page { background: #ffffff; margin: 50px; border: 2px solid #c0c0c0; padding: 10px; }
#header { background: #4b6983; border: 2px solid #7590ae; text-align: center; padding: 10px; color: #ffffff; }
#header h1 { color: #ffffff; }
#body { padding: 10px; }
span.tt { font-family: monospace; }
span.bold { font-weight: bold; }
a:link { text-decoration: none; font-weight: bold; color: #C00; background: #ffc; }
a:visited { text-decoration: none; font-weight: bold; color: #999; background: #ffc; }
a:active { text-decoration: none; font-weight: bold; color: #F00; background: #FC0; }
a:hover { text-decoration: none; color: #C00; background: #FC0; }
</style>
</head>
<body>
<div id="page">
 <div id="header">
 <h1>Group 4 CGI tools</h1>
 </div>
 <div id="body">
  <h2>Bioinformatics tools</h2>
  <h3>Download</h3>
  <form method="get" action="get_pdb.cgi">
	PDB ID: <input type="text" name="pdbid" cols="4" value="1ULI"/><input type="submit" value="Get PDB data"/>
  </form>
  
  <h3>RenderIT</h3>
  <b style="color: #AAF;'">&bdquo;The best quality PDB renderer webservice you can possibly find in this /24 subnet!&rdquo;</b><br/><br/>
  <form method="get" action="render_pymol.cgi">
	PDB ID: <input type="text" name="pdbid" cols="4" value="1ULI"/><input type="submit" value="Render"/>
  </form>
    
  <h3>Import organism genome data into database </h3>
  <form method="get" action="genome_db.cgi">
	Organism (<b>only bacteria!</b>): <input type="text" name="organism" cols="4" value="Mycoplasma genitalium"/><input type="submit" value="Find"/>
  </form>
  
  <h3>Find prosite pattern in database: </h3>
  <form method="get" action="prosite-pattern.cgi">
	Pattern: <input type="text" name="pattern" cols="4" value="P-P-x-x"/><input type="submit" value="Find"/>
  </form>
  
  <h3>Find AC numbers containing a keyword:</h3>
  <form method="get" action="search_keyword.cgi">
	Keyword: <input type="text" name="keyword" cols="4" value="Antigen"/><input type="submit" value="Find"/>
  </form>

  <h3>Get sequence by ID (e.g. AC Number)	:</h3>
  <form method="get" action="get_sequence.cgi">
	ID: <input type="text" name="id" cols="4" value="Q9Y2H6"/><input type="submit" value="Find"/>
  </form>
  
  <h3>Generate SQL Histogram	:</h3>
  <form method="get" action="histo_sql.cgi">
	Upper bound: <input type="text" name="M" value="1000"/><br/>
	Lower bound: <input type="text" name="m" value="100"/><br/>
	Interval: <input type="text" name="c" value="10"/><br/><input type="submit" value="Generate"/>
  </form>
  
    <h3>Find ORFs in sequence</h3>
    <a href="chrI.txt">Test data - chrI of S. cerevisiae</a><br/>
  <form action="orf.cgi" method="post" enctype="multipart/form-data">
	<b>Plaintext</b> sequence file (<b>NOT FASTA!</b>):<input type="file" name="sequence"><br/>
	<input type="submit" value="Find ORFs"/>
  </form>
  
<h3>Find atom contacts in proteins</h3>
  <form action="sscc.cgi" method="get">
	PDBID: <input type="text" name="pdbid" value="3H9U"><br/>
	Contact distance: <input type="text" name="contactdistance" value="5"><br/>
	Atom type: <input type="text" name="atomtype" value="C"><br/>
	Local contact sequence distance: <input type="text" name="localcontactsequencedistance" value="5"><br/>
	<input type="submit" value="Find"/>
  </form>
  
  <h3>Find <a href="http://tardis.nibio.go.jp/homstrad/">HOMSTRAD</a> info</h3>
  <form method="get" action="homstrad.cgi">
	Find single PDB ID: <input type="text" name="pdbid" cols="4" value="1BGX"/><input type="submit" value="Find"/>
  </form>
    <form method="get" action="homstrad_pair.cgi">
	Show alignment of: <input type="text" name="pdbid1" cols="4" value="1BGXT"/> and <input type="text" name="pdbid2" cols="4" value="1BGX"/> <input type="submit" value="Find"/>
  </form>
  
  <h3>BLAST a sequence in a self-supplied database</h3>
  <b>Test data:</b>&nbsp;<a href="mgenitalium-proteome.fa">FASTA M. Genitalium complete proteome</a>&nbsp;|&nbsp;<a href="human-atp7b-coding-seq.fa">Human ATP7b coding sequence</a>&nbsp;|&nbsp;<a href="P00846.fasta"> ATP synthase subunit a (H Sapiens)</a><br/>
  <form action="blastsearch.cgi" method="post" enctype="multipart/form-data">
	<b>Query</b> is a: <input type="radio" name="type" value="prot" checked> Protein sequence<input type="radio" name="type" value="nucl"> Nucleotide sequence<br/>
	Protein Database in FASTA format: <input type="file" name="database"><br/>
	Query sequence (FASTA):<br/> <textarea name="query" cols="50" rows="20"></textarea><br/>
	<input type="submit" value="Query">
  </form>
</body>
</html>
